docs/superpowers/plans/2026-05-14-issue-228-r-cell-parity.md

```
@@ -0,0 +1,636 @@
+# Setup ----
+helper <- function() 1
+# %% transform
+mtcars |> mutate(x = helper())
+```
+
+In the example above the `load` cell ends at `# Setup ----`. The `helper <- function() 1` line is orphan — it belongs to neither cell.
+
+### Active-cell border
+
+The cell containing the cursor gets a top and bottom border so you can see at a glance which cell `Run Current Chunk` will run. Turn it off with `raven.chunks.activeCellIndicator: false`. The colors are themable via `raven.chunk.activeCellBorderTop` and `raven.chunk.activeCellBorderBottom`.
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)